packages/web/src/new/services/FeatureFlags/index.ts

```diff
@@ -1,2 +1 @@
 export * from './FeatureFlags';
-export * from './features';
```

Comment:
why removed?

Reply:
Because there were circular imports in RemoteConfig and FeatureFlags

https://github.com/p2p-org/p2p-wallet-web/blob/feat/PWN-4977/remote_config_api_endpoints/packages/web/src/new/services/RemoteConfig/RemoteConfig.ts#L7

https://github.com/p2p-org/p2p-wallet-web/blob/feat/PWN-4977/remote_config_api_endpoints/packages/web/src/new/services/FeatureFlags/FeatureFlags.ts#L5

RemoteConfig uses Features for preparing featureFlags object from config data

https://github.com/p2p-org/p2p-wallet-web/blob/b24d5ecfbe80d2d9199b114fd68ad86f17ba6876/packages/web/src/new/services/RemoteConfig/RemoteConfig.ts#L47